### PR TITLE
feat(mock-ase): call setFee during seed script

### LINKED ISSUE #1641: Update Mock ASE to use new quoting mechanism
## Context

Fees are now configured within Rafiki. The quote endpoint becomes unnecessary. On start, the mock ASE should properly configure its fees using the new `updateFees` admin endpoint

## Todo
- [ ] remove `/quote` endpoint
- [ ] update `run_seed` script to call `updateFees` admin endpoint
- [ ] update docs

### PR DESCRIPTION
**Pull request by njlie**
## Changes proposed in this pull request

- Remove the `quotes` endpoint from the mock ASE
- Call `setFee` to set the sending fee for each asset on a mock ASE during the seed script.

## Context

Fixes #1641.

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

**Review by BlairCurrey — COMMENTED** [review]
It looks good to me, although I think we shouId probably wait until this is merged: https://github.com/interledger/rafiki/pull/1755

Create quote still relies on the quote endpoint which is removed in this PR, so the quote create fails. Noticed when doing eCommerce flow.

**Comment by sabineschaller** [comment]
Once merge conflicts are resolved, I'll give this another spin.

File: localenv/mock-account-servicing-entity/app/lib/parse_config.server.ts
Comment (mkurapov) [inline localenv/mock-account-servicing-entity/app/lib/parse_config.server.ts:47]: If this is the interface from the seed file, I'm not sure if we need `asset` & `scale`?

Comment (mkurapov) [reply localenv/mock-account-servicing-entity/app/lib/parse_config.server.ts:47]: Seems like we can also remove the `interface Fees` just above

**Review by mkurapov — DISMISSED** [review]

**Review by sabineschaller — APPROVED** [review]
I gave it a spin and it works. Thank you!

**Pull request merged**
